### DISCUSSION TITLE
Enable Tab Panel Auto-Show on Hover for Drag-and-Drop Between Windows in Compact Mode

### DISCUSSION THREAD

**Discussion by ZombieLurker** [zen-browser/desktop#2993] [Ideas]
## Feature Request

Not sure if this matters, I assume not. But this behavior is found in the Arch Linux AUR 'zen-browser-avx2-bin' package.

### Current Behavior
- When Zen-Browser is in compact mode with the tab panel hidden, the tab panel doesn’t auto-show on mouse hover when dragging a tab from one browser window to another.

### Desired Behavior
- It would be helpful if, during tab drag-and-drop in compact mode, the tab panel could appear automatically on mouse hover, allowing users to drop the tab into another Zen-Browser window without extra steps.

**Comment by spslabo** [comment]
This feature would be very helpful or maybe an option to move to a different window/workspace on the context menu. I have to disable compact mode to be able to drag a tab from one window to another. 
I am also on Arch using [zen-browser-bin ](https://aur.archlinux.org/packages/zen-browser-bin)	1.6b-4

**Comment by Khang5687** [comment]
I agree, I frequently use drag-and-drop search. Highlighting the text, then just drag it to a new tab. But with compact mode, I cannot do it.

**Comment by spslabo** [comment]
I believe this has been fixed/added now, I can drag and drop tabs from one window to another with compact mode. Zen 1.10.1b (Firefox 136.0.2) (64-bit)
